duckduckgo/Android
# Add privacy practices detail screen

Asana Issue URL: https://app.asana.com/0/488551667048375/471193197990042

## Description
Displays privacy practices details

## Steps to Test this PR:
**TEST ONE:**
1. Visit google.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as poor and poor practices should be shown

**TEST TWO:**
1. Visit duckduckgo.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as good and good practice should be shown

**TEST THREE:**
1. Visit netflix.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as mixed and mixed practices should be shown

**TEST FOUR:**
1. Visit cnn.com
1. Open the Privacy Dashboard and tap open Privacy Practices
1. Practices should show as unknown and no practices should be shown (we will later add an additional message in the blank space)

**TEST FIVE**
Following from the above test, tap the TOSD panel at the bottom of the screen and ensure that it launches the tosdr url
